src/core/src/bootstrap/EnvLayer.py

```diff
@@ -419,7 +419,7 @@ def __write_record(self, operation, code, output, delay, timestamp=None):
 
         try:
             record = {
-                "timestamp": str(timestamp) if timestamp is not None else datetime.datetime.strptime(str(datetime.datetime.utcnow()).split(".")[0], "%Y-%m-%dT%H:%M:%SZ"), #WRONG
+                "timestamp": str(timestamp) if timestamp is not None else datetime.datetime.strptime(str(datetime.datetime.utcnow()).split(".")[0], Constants.UTC_DATETIME_FORMAT), #WRONG
```

Comment:
In a different PR (specifically for this), please apply this constant wherever applicable elsewhere.